src/components/Controls.vue

```diff
@@ -54,6 +56,13 @@ export default {
       } else {
         return []
       }
+    },
+    linkVariableList() {
+      if (typeof this.graphStructure.links[0] !== 'undefined') {
+        return Object.keys(this.graphStructure.links[0]).concat(["table", null]) 
+      } else {
+        return []
+      }
```

Comment:
It occurs to me that we may also want to allow the user to set width/color of the links based on data residing on the source or target node. That's something to explore in a future PR. But by the same token, I'm not sure it makes sense to offer `table` as an option here, since currently we only allow a single edge table per graph, nor `source` and `target` by themselves, since those turn out to just be IDs for the nodes connected at either end.

Reply:
Opened #58 for the first part of your comment. I'll remove table as an option and source and target

Reply:
Fixed in 2a77ea5